Updating the featured packages audit-helper -> DPE

b-per opened the pull request:
Updating the featured packages to reflect some of the dbt-labs repos with higher star counts.

### 🎩 

#### Before

<img width="1244" alt="image" src="https://github.com/dbt-labs/hub.getdbt.com/assets/44704949/643b697a-9728-47a0-98f5-db2a3eb01364">

#### After

<img width="1278" alt="image" src="https://github.com/dbt-labs/hub.getdbt.com/assets/44704949/d37afd15-2754-4d14-afcb-3684c35c2b99">

dbeatty10 commented:
Nice @b-per -- thanks for instigating a review of the featured packages!

### Problem to solve
Something that came up that I'd like to solve...

In my browser, it's rending like this by default:
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/44704949/222493570-ff51cfed-1e40-4520-9f8a-32fcec850922.png">

If I zoom out one level, it has much nicer rendering and gets them all on one row:
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/44704949/222493736-931def02-aa7e-49b4-8816-0a57a00b1102.png">

It's not all about me -- but if I'm running into it, I can only assume others will also.

### Some ideas

0. Live with it or buy a larger monitor
1. Adjust the CSS (or whatever) to make the font smaller by default, so it can fit on one row
2. Add audit helper as a 4th item on the list, so it will display a 2X2 grid for display sizes like mine
3. Rename `dbt_project_evaluator` so that it is 12 characters or less (okay, just kidding about this one)

It seems like Idea 1 is the way to go, but open to a wide variety of ideas of how to solve this. What do you think?

b-per commented:
@dbeatty10 reviving this one!

I added another row of 3 featured packages, having a total of 6 now! This number feels reasonable to me knowing the high number of packages now available on the hub.

I have also slightly updated the style of the cards:
- I removed the "dbt models for ..." which seemed irrelevant for most packages as they don't contain models but macros
- I made the icon and the text smaller

Let me know what you think!!!

dbeatty10 reviewed (approved):
Amazing @b-per 👑
